test/e2e/components/vl-map-search.js

```diff
@@ -21,26 +24,31 @@ class VlMapSearch extends VlElement {
   async search(text) {
     const search = await this._getSearch();
     const input = await search.findElement(By.css('.vl-select__list > input'));
+    await this.driver.executeScript(`arguments[0].focus()`, input);
     await this.driver.executeScript(`arguments[0].value='${text}'`, input);
+    await this.driver.executeScript(`arguments[0].dispatchEvent(new CustomEvent('keyup', {composed: true, bubbles: true}))`, input);
```

Comment:
encapsuleren !

Reply:
dit lijkt in vl-search te moeten zitten?

Reply:
ja, maar als we de select gebruiken zoals in de e2e testen van vl-search krijgen we hier een foutmelding dat het element niet interactable is (of gelijkaardig, weet het niet meer vanbuiten).
Via deze weg lukte het ons wel om iets te zoeken in de map.
@coemans, klopt het ongeveer wat ik zeg? Moet ik hier nog iets veranderen?

Reply:
Deze code is geïntroduceerd naar aanleiding van een Selenium Webdriver API Firefox bug, waardoor de input niet rechtstreeks door selenium kan opgevuld worden met de send keys functie. In afwachting tot de bug fix, zou ik de fix op deze locatie behouden. Ik heb liever dat in de vl-ui-select wel de send keys gebruikt wordt omdat het meer aanleunt bij de interactie van een eindgebruiker.

Encapsuleren naar een protected/private methode kan hier wel nog uitgevoerd worden.